contracts/VToken.sol

```diff
@@ -1786,6 +1937,48 @@ contract VToken is Ownable2StepUpgradeable, VTokenInterface, ExponentialNoError,
         emit NewAccessControlManager(oldAccessControlManager, accessControlManager);
     }
 
+    /**
+     * @notice Sets the utilization threshold for stable rate rebalancing
+     * @param utilizationRateThreshold The utilization rate threshold
+     * @custom:access Controlled by AccessControlManager
+     */
+    function setRebalanceUtilizationRateThreshold(uint256 utilizationRateThreshold) external {
+        bool canCallFunction = AccessControlManager(accessControlManager).isAllowedToCall(
+            msg.sender,
+            "setRebalanceUtilizationRateThreshold(uint256)"
+        );
+        // Check caller is allowed to call this function
+        if (!canCallFunction) {
+            revert SetRebalanceUtilizationRateThresholdAdminCheck();
+        }
```

Comment:
We could encapsulate this logic in a function, as we are doing in the new Comptroller:

```
    /// @notice Reverts if the call is not allowed by AccessControlManager
    /// @param signature Method signature
    function _checkAccessAllowed(string memory signature) internal view {
        bool isAllowedToCall = AccessControlManager(accessControl).isAllowedToCall(msg.sender, signature);

        if (!isAllowedToCall) {
            revert Unauthorized(msg.sender, address(this), signature);
        }
    }
```

I think we don't need specific errors.

Reply:
Yes, sounds better, and we can also pass the error as the argument(If we want to use different errors).
Can we create a seperate ticket for that?

Reply:
I would do it the same way we are doing it in IL, not using specific errors but the generic `Unauthorized`

Reply:
VEN-1123